test/CDB_QuotaTest_expect

```diff
@@ -9,7 +9,7 @@ INSERT 0 2048
 INSERT 0 2048
 INSERT 0 2048
 2
-ERROR:  Quota exceeded by 567.998046875KB
+ERROR:  Quota exceeded by 519.998046875KB
 0
```

Comment:
Well, I think it is normal since the table has not the timestamp columns it takes less storage space.